app/_data/docs_nav_gateway_3.6.x.yml

```diff
@@ -447,7 +447,7 @@ items:
         url: /plugin-development/admin-api
       - text: Writing Tests
         url: /plugin-development/tests
-      - text: (un)Installing your Plugin
+      - text: Managing your custom plugin
```

Comment:
IMO "Managing" is too generic.  I would suggest "Deploying" as that seems relevant to the general topic here.